app/src/app/pages/plando-page/plando-items/plando-items.component.ts

```diff
@@ -123,6 +125,12 @@ export class PlandoItemsComponent {
   }
 
   private _filter(initialOptions: Array<string>, value: string): void {
+    if (value.startsWith('TRAP')) {
```

Comment:
Could do
`if (value.toLowerCase().startsWith('trap')) {`
instead so when you type "trap" all the trap suggestions still appear

Reply:
Good call. Added an entry for `TRAP` to the trap autofill as well.